WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -0,0 +1,34 @@
+// Add params from the search fields to the bootstrap-table for on Ajax request.
+function queryParams(params) {
```

Comment:
Unfortunately it must be global accessible because as I've just checked bootstrap-table looks for it inside the window object.